configs/local-dev.yaml

```diff
@@ -68,8 +68,11 @@ astronomer:
       requests:
         cpu: "0m"
         memory: "0Mi"
-
     config:
+      registry:
+        protectedCustomRegistryEnabled: true
+      deployments:
+        enableUpdateDeploymentImageEndpoint: true
       emailConfirmation: false
```

Comment:
Do we need the above 4 lines? Is this a dev only set up which should get reflected in our dev cluster?

Reply:
not sure why this was added.. but ill just close the issue and just update the versions..